src/ess/reduce/nexus/workflow.py

```diff
@@ -647,32 +648,21 @@ def LoadDetectorWorkflow() -> sciline.Pipeline:
     return wf
 
 
-def GenericNeXusWorkflow(
-    *,
-    run_types: Sequence[sciline.typing.Key] | None = None,
-    monitor_types: Sequence[sciline.typing.Key] | None = None,
-) -> sciline.Pipeline:
+def GenericNeXusWorkflow() -> sciline.Pipeline:
```

Comment:
This change will break downstream such as ESSsans which is using these args already. Is this deliberate?

Reply:
Deliberate in that it is easier than keeping the arguments in two places.

But also, then it breaks, the error should indicate to us that the code should be changed. As it stand, it suffers from the exact problem that I am trying to fix.

Note that I converted this to a draft because it does not actually work with a full workflow. I am adding a way to specify target types instead of hard coding `DetectorData` and `MonitorData`.